src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,15 +1050,14 @@ def readCondorLog(self, job):
             cres = condor.read_events(logfileobj, 1)
             ulog = list(cres)
```

Comment:
Well yes - in fact, I would have expected to be even slower than that for iterating through a list of 10M integers.

However, in the above example, we don't have a list of events - each iteration through the loop is a call to a HTCondor library.  The "optimization" from the for loop means that we don't keep all the events in memory - just the last one.